CONTRIBUTING.md

```diff
@@ -0,0 +1,614 @@
+
+extension  
+: Do not use. Use *plugin* instead.
+
+execute/executed  
+: Do not use. Use *run* (or *ran* for past-tense) instead.
+
+front end, front-end, frontend  
+: Do not use frontend. Use *front end* as a noun. Use front-end as a compound adjective.
+
+HTML  
+: Do not spell out. Do not use *html* (except in codeblocks).
+
+JavaScript, JS  
+: The correct capitalization is *JavaScript*. Do not use *Javascript* or *javascript*. Do not use *JS* unless it is referring to a specific library such as *PrismJS*
+
+For example: Another popular JavaScript minification tool, UglifyJS2, will corrupt TinyMCE unless ASCII-only is set.
+
+menu bar, menubar  
+: Use *menu bar* as two words. Do not use *menubar* or *menu-bar* as a compound word.
+
+plugin  
+: The preferred way of referring to an add-on or extension for TinyMCE. Do not use *add-on*, *extension*, *app*, or *plug-in*.
+
+premium features  
+: *premium features* and not *Premium features*.
+
+programmer  
+: Do not use. Use *developer*.
+
+question marks ("?")
+: Use questions sparingly. In general, user want us to give them answers. When a customer needs to make a decision, a question *may be* appropriate.
+
+*Bad example:*
+
+Why make TinyMCE mobile friendly?
+
+*Good examples:*
+
+- If you forgot your password, provide your secret answer.
+
+- More nature themes are available online.
+
+- Do you want to save your changes?
+
+quickstart, quick start
+: Content to help developers get our products up and running quickly. Use *quickstart*.
+
+rich text editor, rich-text editor, RTE
+: Most common way to refer to TinyMCE’s product category.
+
+Use rich text editor rather than rich-text editor (even if the later may be more technically correct). Do not use RTE.
+
+screen size, screensize
+: Use *screen size*.
+
+SDK
+: Do not use. Use *Self-hosted installation*.
+
+See or Refer to…
+: Use *see* for internal links, use *visit* for external links.
+
+Such as: For information on creating links, see: &lt;link&gt;.
+
+self-hosted
+: The preferred way to refer to an on-premises deployment of our technology.
+
+self-hosted premium plugins
+: Preferred. Do not use *premium plugin SDKs*.
+
+should or must
+: Avoid using *must* or *should*. Only use where it is unavoidable.
+
+submenu
+: Use *submenu*. Do not use *sub-menu* or *sub menu*.
+
+Software as a Service (SaaS)
+: Spell out on the first mention in a section for general audience. For a technical audience, it is always OK to use SaaS.
+
+Do not capitalize as SAAS.
+
+Do not hyphenate as a modifier, but include the abbreviated form in parentheses.
+
+spell checking, spell checker, spell check
+: Do not use the compound words *spellcheck* or *spellchecker*
+
+Do not hyphenate.
+
+textarea, text box, textbox
+: Use *textarea* to refer to a multi-line text box in a form.
+
+Tiny
+: Use to refer to the company or our entire solution/platform.
+
+Do not use *Tiny* to refer to the core editor, which is *TinyMCE*.
+
+Tiny 4.x, Tiny 5.0
```

Comment:
This and lines 559, 562 and 172 are the minor differences compared to the CONTRIBUTING file for 6 doc